dpnp/__init__.py

```diff
@@ -77,6 +74,81 @@
 from . import scipy as scipy
 
 
+# =============================================================================
+# Data types, constants and type-related helpers
+# =============================================================================
+
+# -----------------------------------------------------------------------------
+# Data types (borrowed from NumPy)
+#
+# The order of these declarations are borrowed from the NumPy document:
+# https://numpy.org/doc/stable/reference/arrays.scalars.html
```

Comment:
It does not really make any sense, because there is a single `from .dpnp_iface_types import` below which is connected with that.

Reply:
The order is defined by the python sort linter and not by the pointing numpy's page.